### ISSUE TITLE
SQL Errors "1305" on upgrading from 10.0.7 to 10.0.8

### ISSUE THREAD

**Issue by prbt2016** [glpi-project/glpi#15141]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

Upgrading from 10.0.7 to 10.0.8

### Bug description

Hello,

I was in the process of upgrading GLPI from 10.0.7 to 10.0.8 on Centos7 with PHP 7.4, MYSQL 5.7, Apache 2.2. 

However , I encountered an issue while doing so . Getting the following errors post upgrade i.e : 

```
SQL Error "1305": FUNCTION glpi788.REGEXP_SUBSTR does not exist in query "UPDATE `glpi_computervirtualmachines` SET `ram` = REGEXP_SUBSTR(`ram`, '[0-9]+') WHERE `ram` REGEXP '[^0-9]+'"

SQL Error "1305": FUNCTION glpi788.REGEXP_SUBSTR does not exist in query "UPDATE `glpi_networkequipments` SET `ram` = REGEXP_SUBSTR(`ram`, '[0-9]+') WHERE `ram` REGEXP '[^0-9]+'"

```

Following is the screenshot of the same  : 

![image](https://github.com/glpi-project/glpi/assets/17871448/73e0b03f-2c44-4a44-ab54-3dc16c8d9617)

What could be the issue?

### Relevant log output

```shell
[2023-07-06 08:51:29] glpisqllog.ERROR: DBmysql::query() in {{PATH}}/src/DBmysql.php line 379
  *** MySQL query error:
  SQL: UPDATE `glpi_computervirtualmachines` SET `ram` = REGEXP_SUBSTR(`ram`, '[0-9]+') WHERE `ram` REGEXP '[^0-9]+'
  Error: FUNCTION glpi788.REGEXP_SUBSTR does not exist
  Backtrace :
  src/DBmysql.php:1468                               
  ...ations/update_10.0.7_to_10.0.8/ram_field.php:59 DBmysql->update()
  install/migrations/update_10.0.7_to_10.0.8.php:59  require()
  src/Update.php:269                                 update1007to1008()
  install/update.php:143                             Update->doUpdates()
  install/update.php:252                             doUpdateDb()
  {"user":"@{{DOMAIN}}"} 
[2023-07-06 08:51:29] glpisqllog.ERROR: DBmysql::query() in {{PATH}}/src/DBmysql.php line 379
  *** MySQL query error:
  SQL: UPDATE `glpi_networkequipments` SET `ram` = REGEXP_SUBSTR(`ram`, '[0-9]+') WHERE `ram` REGEXP '[^0-9]+'
  Error: FUNCTION glpi788.REGEXP_SUBSTR does not exist
  Backtrace :
  src/DBmysql.php:1468                               
  ...ations/update_10.0.7_to_10.0.8/ram_field.php:59 DBmysql->update()
  install/migrations/update_10.0.7_to_10.0.8.php:59  require()
  src/Update.php:269                                 update1007to1008()
  install/update.php:143                             Update->doUpdates()
  install/update.php:252                             doUpdateDb()
  {"user":"@{{DOMAIN}}","mem_usage":"0.022\", 17.67Mio)"}
```

### Steps To reproduce

1. Upgrade GLPI from 10.0.7 to latest 10.0.8.
2. Click upgrade.
3. SQL error 1305 thrown

**Comment by trasher** [comment]
Fixed, see https://github.com/glpi-project/glpi/pull/15127

**Comment by prbt2016** [comment]
Hello @trasher,

Thanks  for looking into the issue. Any ETA for 10.0.9 and I can see the fix under 10.0.9 milestone.

Is it okay if I use #15127 ?

Kindly let me know.
